Lib/test/test_pprint.py

```diff
@@ -346,6 +346,36 @@ def test_mapping_proxy(self):
                           ('lazy', 7),
                           ('dog', 8)]))""")
 
+    def test_empty_simple_namespace(self):
+        self.assertEqual(pprint.pformat(types.SimpleNamespace()), "namespace()")
```

Comment:
FWIW, code is almost always more readable when limited to one action per line.  Among other things, it makes it much easier to understand what's happening (especially when parentheses are involved).

```suggestion
        formatted = pprint.pformat(
            types.SimpleNamespace())
        self.assertEqual(formatted, "namespace()")
```

*Personally* I like to separate the 3 parts (setup, actual-thing-to-test, assertions) with blank lines, for further readability, but that's more personal preference:

```suggestion
        ns = types.SimpleNamespace()
        
        formatted = pprint.pformat(ns)
        
        self.assertEqual(formatted, "namespace()")
```

Reply:
I do not think that adding an empty line after every line of code will make the code more readable. Empty lines should be used for separating logical sections of the code.

This code is consistent with the code in this file and looks clear to me. I do not think that any changes are needed.

Reply:
I followed most of @ericsnowcurrently's suggestions, but without the empty lines and without `textwrap.dedent` to be consistent with the rest of the file and I think it has made the test more clear.